packages/dashboard-v2/src/pages/auth/signup.js

```diff
@@ -1,10 +1,32 @@
 import { useEffect, useState } from "react";
+import { navigate } from "gatsby";
 
 import AuthLayout from "../../layouts/AuthLayout";
 
+import { Alert } from "../../components/Alert";
 import HighlightedLink from "../../components/HighlightedLink";
 import { SignUpForm } from "../../components/forms/SignUpForm";
-import { navigate } from "gatsby";
+import { usePortalSettings } from "../../contexts/portal-settings";
+
+const FreePortalHeader = () => (
+  <div className="mt-4 mb-8 font-sans">
+    <h3>Create your free account</h3>
+    <p className="font-light text-lg">Includes 100 GB storage at basic speed</p>
```

Comment:
this should be dynamic based on /limits since we might change the storage for free account